homeassistant/__main__.py

```diff
@@ -358,23 +309,13 @@ def main():
 
     args = get_arguments()
 
+    if args.script is not None:
+        from homeassistant import scripts
+        return scripts.run(args.script)
```

Comment:
If there are two levels of 'argparse', here and in the script, wont duplicate --args be interpreted by the first argparse and never be passed to script.run?

(i.e -c or --config in the script)